js/client.js

```diff
@@ -2509,13 +2509,13 @@ function toId() {
 			var name = data.name;
 			var avatar = data.avatar || '';
 			var group = ((Config.groups[data.roomGroup] || {}).name || '');
-			var globalgroup = ((Config.groups[(data.group || Config.defaultGroup || ' ')] || {}).name || '');
-			if (globalgroup) {
-				if (!group || group === globalgroup) {
-					group = "Global " + globalgroup;
-					globalgroup = '';
+			var globalGroupObject = (Config.groups[(data.group || Config.defaultGroup || ' ')] || null);
```

Comment:
```suggestion
			var globalGroupObject = (Config.groups[data.group || Config.defaultGroup || ' '] || null);
```

Reply:
tbh I don't really like `Object` as part of a variable name. Probably call these `globalGroup` and `globalGroupName` instead of `globalGroupObject` and `globalgroup`?